### PR TITLE
[sentry-native] update to 0.7.2.

### PR DESCRIPTION
**Pull request by AenBleidd**
Added optional 'compression' parameter.

### CODE REVIEW

File: ports/sentry-native/vcpkg.json [hunk 1]
```diff
@@ -44,6 +44,18 @@
         }
       ]
     },
+    "compression": {
+      "description": [
+        "Enables gzip compression support for the transport.",
+        "Cf.https://github.com/getsentry/sentry-native#compile-time-options"
```
Comment (BillyONeal) [inline ports/sentry-native/vcpkg.json:50]: The setting does not appear to be listed here?

File: ports/sentry-native/vcpkg.json [hunk 2]
```diff
@@ -44,6 +44,18 @@
...
+      ],
+      "dependencies": [
+        {
+          "name": "zlib",
+          "platform": "!android & !ios"
```
Comment (BillyONeal) [inline ports/sentry-native/vcpkg.json:55]: This appears to be required on android and ios: https://github.com/getsentry/sentry-native/blob/55b6761837ebea791e31724b97b1560cda3b9244/CMakeLists.txt#L288

Can you explain why the dependency isn't listed here?

Thanks!

File: ports/sentry-native/vcpkg.json (hunk 1 above)
Comment (AenBleidd) [reply ports/sentry-native/vcpkg.json:50]: Looks like they just forgot to add it there. Will let them know about this.

File: ports/sentry-native/vcpkg.json (hunk 2 above)
Comment (AenBleidd) [reply ports/sentry-native/vcpkg.json:55]: I have double checked this, and you're completely right! We can definitely use this for android and ios.

File: ports/sentry-native/vcpkg.json (hunk 1 above)
Comment (AenBleidd) [reply ports/sentry-native/vcpkg.json:50]: https://github.com/getsentry/sentry-native/pull/976

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**